tests/models/test_hooks.py

```diff
@@ -229,26 +229,6 @@ def train_dataloader(self):
     trainer.fit(model)
 
 
-@pytest.mark.parametrize('max_epochs,batch_idx_', [(2, 5), (3, 8), (4, 12)])
```

Comment:
Moved the test since it is testing more of the training loop internals than the model hook itself.